OTel, metrics, heap dump, use 3.1.0 (#186)
SteeltoeOSS/Documentation: api/v3/bootstrap/index.md

# Application Bootstrapping

> A new experiment in improving the Steeltoe developer experience has been included in Steeltoe 3.1.0 that allows the configuration of most Steeltoe components with a single line of code in your application. The package is named [`Steeltoe.Bootstrap.Autoconfig`](https://github.com/SteeltoeOSS/Steeltoe/tree/main/src/Bootstrap/src/Autoconfig), and it works by applying the same extensions that are already included in Steeltoe packages to automatically wire up each of those components.

Reviewer: ```suggestion
In order to improve the Steeltoe developer experience, Steeltoe 3.1.0 includes this new feature that allows the configuration of most Steeltoe components with a single line of code in your application. The package is named [`Steeltoe.Bootstrap.Autoconfig`](https://github.com/SteeltoeOSS/Steeltoe/tree/main/src/Bootstrap/src/Autoconfig), and it works by applying the same extensions that are already included in Steeltoe packages to automatically wire up each of those components.
```
